packages/api-generator/src/maps/v-data-table.js

```diff
@@ -83,6 +83,7 @@ const DataTableItemColumnScopedProps = {
   item: 'any',
   header: 'TableHeader',
   value: 'any',
+  index: 'number',
```

Comment:
it should be added also to `DataTableItemScopedProps` and `DataTableExpandedItemScopedProps` (any others?)